spec/order_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'order'
+
+describe Order do
+  let(:time_klass){ double(:time_klass, new: 0) }
+  subject(:order) { Order.new(time_klass) }
+
+  let(:quantity){ 3 }
+  let(:price) { 1 }
+  let(:total) { quantity * price }
+  let(:name) { 'Food' }
+  let(:dish){ double(:dish, name: name, price: price)}
+  let(:receipt){ "Your order has been placed and will be delivered at 3600: #{name} x#{quantity} = £#{price * quantity}. Total: #{total}" }
```

Comment:
Metrics/LineLength: Line is too long. [139/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)